### ISSUE TITLE
Scrollable code blocks not accessible by keyboard

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-design-system#1343]
Keyboard users are currently unable to view code blocks using standard keyboard commands. This was raised in a DAC audit - they have suggested applying `tabindex=0` to the code blocks.

Issue raised by @kr8n3r via Zendesk:

> Although this may be a hinderance to screen reader users navigating out of context via the TAB key, this is preferable to the element being inaccessible to keyboard only users who would not be able to view the code extract presented beyond the width of the container.

> Screen reader users have the option to navigate the page content in a variety of ways and applying a tabindex="0" to scrollable regions should not prevent them from accessing that content. Where as keyboard only users are limited to fewer methods of navigation and are currently unable to view the code extract at all using standard keyboard commands.

> codepen example https://codepen.io/kr8n3r/pen/vYGxgOY
> once focussed allows for arrow left/right scroll

**Comment by hannalaakso** [comment]
This is also related to https://github.com/alphagov/govuk-design-system/issues/1336
